### ISSUE TITLE
Documentation of removal procedure

### ISSUE THREAD

**Issue by 73** [balbusm/xul-ext-eds-calendar#32]
If the extension is removed, the entries are not. 
Where shall the entries be removed from?

**Comment by balbusm** [comment]
Why would anyone remove this addon :)?
Could you tell why you want to remove this addon?

Addon copies all your calendar entries to Evolution storage.
Having said that you have 2 options to remove entries:
# Option 1 
- Install addon
- Go to calendar view in Thunderbird
- In left panel uncheck all your calendars
- Uninstall addon
- It might be required to restart computer so system calendar will pick up changes
# Option 2
- Install Evolution
- Go to calendar view (don't need to setup your account)
- Remove all your calendars
- It might be required to restart computer so system calendar will pick up changes

**Comment by 73** [comment]
Ah. I already worked thru Option 2 (beside the reboot, and a view entries
remained). We will how this goes after reboot. I think it is important to
write that into the readme, because after the user reverted the
installation the calenders remain. That might be unexpected if you don't
know how this plugin works.

I installed the add-on because I am a calendar junky (why else would I use
the add-on in the first place?), And therefore I have around 10 calenders
in Thunderbird. They all got dumped into the little shell overview. That
did not help. Additionally, all alerts/notification/remainders (whatever
they are called) yield in two popups.

I would like to get a good agenda view for today's calendar entries, but
Gnome-Shell does not provide one. The plug-in did as promised.
